.github/workflows/pr-metadata.yml

```diff
@@ -1,15 +1,17 @@
 name: Jira Report
 
 on:
-  # This should probably be pull_request_target.
-  pull_request:
+  # Having this be pull_request_target rather than pull_request means it runs in the context of the
+  # target branch rather than the PR, which in turn means the checkout is of the target.
+  pull_request_target:
 
 jobs:
   example_comment_pr:
     runs-on: ubuntu-latest
     name: Report Jira data to PR comment
     env:
       PR_TITLE: ${{ github.event.pull_request.title }}
+      PR_NUMBER: ${{ github.event.pull_request.number }}
     steps:
       - name: Checkout
         uses: actions/checkout@v3
```

Comment:
Given this is doing a checkout of the target branch, not the PR I wonder if we should tell it to checkout master?  Then we'd have the ability to change "rules" for release branches without doing backports.